lib/services/notifications.js

```diff
@@ -30,7 +32,7 @@ function callWebhook(event, url, data) {
   }
 
   return rest.fetch(url, options).then(function (response) {
-    log('info', 'called webhook', _.pick(response, ['status', 'statusText']));
+    log('info', `called webhook ${_.pick(response, ['status', 'statusText'])}`);
```

Comment:
do we need to log allllll resp statuses?

Reply:
It only logs `status` and `statusText`

Reply:
i know, but would it be more worthwhile to log non-200 resps?